### ISSUE TITLE
Provide new API endpoints for search-indexer data

### ISSUE THREAD

**Issue by wileyj** [blockstack/blockstack-core#901]
please create 2 new `api` endpoints for the search-index data:
proposed :
- `/api/v1/index/profile-data` pointing to file located at: https://storage.googleapis.com/blockstack-search_indexer_data/profile_data.json
- `/api/v1/index/blockchain-data` pointing to file located at: https://storage.googleapis.com/blockstack-search_indexer_data/blockchain_data.json
files are configured to  be updated every hour at `00` minutes if changes are detected based on the md5 checksum of the local file compared with the files in the links above.

**Comment by moxiegirl** [comment]
Original discussion: https://forum.blockstack.org/t/rfc-public-blockstack-core-indexing/7117/10 . Feel free to continue discussion on this issue.

**Comment by kantai** [comment]
Okay -- how about this? A single endpoint which returns a JSON:

```
 { indexes : { profileData: "https://url", blockchainData: "url" } }
```

And the `indexes` key is `false` in the event that the urls aren't configured on that particular deployment (which will be default).

The index URLs will be configured via ENV VARS:

```
BSK_API_PROFILE_URL
BSK_API_BLOCKCHAIN_URL
```

Does that make sense to you @wileyj ?

**Comment by kantai** [comment]
This was resolved in #909
